src/components/List/SimpleList/SimpleList.jsx

```diff
@@ -51,6 +51,12 @@ const propTypes = {
   pageSize: PropTypes.string,
   /** callback function returned a modified list */
   onListUpdated: PropTypes.func,
+  /** optionally renders SimplePagination at the bottom of the list */
+  showPagination: PropTypes.bool,
```

Comment:
In HierarchyList, Table, and a few other components this is typically named `hasPagination`. 
```suggestion
  hasPagination: PropTypes.bool,
```